assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm

```diff
@@ -102,7 +102,7 @@
 						onclick="ASN.invokeDownloadUrl('$accessPointUrl', '$actionString', '$alertMessage', '$contextString', '$viewString', '$searchString', '$showSubmissionByFilterSearchOnly', this);" value="$tlang.getString('downloadall.button.download')" />
 			#else
 				<input type="button" name="uploadButton" id="uploadButton"  accesskey="s" class="active"
-						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action 'doUpload_all')'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
+						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action 'doUpload_all' 'sakai_csrf_token=$validator.escapeUrl($sakai_csrf_token)')'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
```

Comment:
I've tested without two apostrophes and empty screen appears and this error is displayed in log:

ERROR [http-nio-8080-exec-4] velocity.sakai.assignment.tool.log Parser Error: /vm/assignment/chef_assignments_instructor_uploadAll.vm
org.apache.velocity.runtime.directive.MacroParseException: Invalid arg #2 in VM #toolLinkParam/vm/assignment/chef_assignments_instructor_uploadAll.vm[line 105, column 164]